webapps/frontend/app/viewmodels/wizards/createfromtemplatewizard/gather.js

```diff
@@ -66,6 +67,26 @@ define([
                 fields.push('name');
                 reasons.push($.t('ovs:wizards.createft.gather.noname'));
             }
+            if (self.data.vMachinesNames().contains(self.data.name())) {
+                valid = false;
+                fields.push('name');
+                reasons.push($.t('ovs:wizards.createft.gather.duplicatename', {
+                    name: self.data.name()
+                }));
+            }
+            if (self.data.amount() > 1) {
+                var i = self.data.startnr();
+                for (i = self.data.startnr(); i < (self.data.startnr() + self.data.amount()); i++) {
+                    name = self.data.name() + '-' + i;
```

Comment:
Use ```var name, i = self.data.startnr();``` before the for loop, then you don't initialize a new variable every iteration.